docs/source/modules/tutorial.rst

```diff
@@ -95,13 +95,11 @@ various components. They are:
 3. ``candidate_models``: various pre-trained models/models that have already been scored.
 4. ``model-tools``: helper functions that translate from machine learning models
    to brain models to be tested on brain-score.
-5. ``brainio_collection``: the repo that packages and collects the stimuli/data.
-6. ``brainio_base``: repo that contains various data structures for ``BrainIO``.
+6. ``brainio``: the packaging and lookup system for data assemblies and stimulus sets.
```

Comment:
5.? Since we're getting rid of previous 5+6

Reply:
Looks like I succeeded in making the reStructuredText list render right on readthedocs, but my commit comment markdown fails.  I can never remember where GitHub uses markdown.